packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -272,6 +272,32 @@ const fn = new lambda.Function(this, 'MyFunction', {
 See [the AWS documentation](https://docs.aws.amazon.com/lambda/latest/dg/concurrent-executions.html)
 managing concurrency.
 
+### Configuring AutoScaling
+
+You can configure autoscaling of provisioned concurrency on a function alias using the `autoScaleProvisionedConcurrency()` method:
+
+```ts
+const alias = new lambda.Alias(stack, 'Alias', {
+  aliasName: 'prod',
+  version,
+});
+
+// Configure autoscaling to track utilization
+alias.autoScaleProvisionedConcurrency({ minCapacity: 1, maxCapacity: 50 }).scaleOnUtilization({
+  targetUtilizationValue: 0.5,
```

Comment:
```suggestion
  targetUtilization: 0.5,
```

Reply:
changed to `utilizationTarget` per comment down below.